test/integration/features/step_definitions/github-api-steps.js

```diff
@@ -9,19 +9,19 @@ const user = any.simpleObject();
 
 Before(function () {
   nock.disableNetConnect();
-
-  githubScope = nock('https://api.github.com/');
 });
 
 After(() => {
   nock.enableNetConnect();
   nock.cleanAll();
 
-  assert.isTrue(githubScope.isDone());
+  assert.isTrue(githubScope?.isDone() ?? true);
```

Comment:
what is the scenario where `githubScope` would not be defined at this point? is there a way we could make it always defined?